data/campaigns/Descent_Into_Darkness/scenarios/03_A_Haunting_in_Winter.cfg

```diff
@@ -1029,21 +1029,21 @@
                 {VARIABLE_OP malin.experience add 30}
                 [floating_text]
                     x,y=$malin.x,$malin.y
-                    text="<span color='#800080'>" + "+30 exp" + "</span>"
+                    text=_ "<span color='#800080'>" + "+30 exp" + "</span>"
```

Comment:
I think the underscore should instead go after the first `+`, here and for all similar instances (there are quite a few in this file).

Reply:
actually this whole section could probably be simplified so that the same thing isn't repeated 3 times; mind if I do it that way instead?